pkg/gardenlet/controller/seed/seed.go

```diff
@@ -144,26 +157,25 @@ func (c *Controller) Run(ctx context.Context, workers int) {
 	var waitGroup sync.WaitGroup
 
 	if !cache.WaitForCacheSync(ctx.Done(), c.hasSyncedFuncs...) {
-		logger.Logger.Error("Timed out waiting for caches to sync")
+		c.log.Error(wait.ErrWaitTimeout, "Timed out waiting for caches to sync")
 		return
 	}
 
 	// Count number of running workers.
 	go func() {
 		for res := range c.workerCh {
 			c.numberOfRunningWorkers += res
-			logger.Logger.Debugf("Current number of running Seed workers is %d", c.numberOfRunningWorkers)
```

Comment:
This log was removed intentionally I guess?

Reply:
Yes, we removed such logs in all other similar refactoring PRs as well. Soon, when we adopt the `controller-runtime`, these go routines will disappear completely anyways.